modules/Api/Controllers/PostController.php

```diff
@@ -125,7 +153,22 @@ public function amendPost(Request $request, Response $response, int $id)
 
         $post = $this->postService->getPost($id);
 
-        if (!$post) {
+        if (!empty($post) && $post['user_uuid'] == auth()->user()->getData()['uuid']){
```

Comment:
instead of auth()->user()->getData()['uuid']) use auth()->user()->getFieldValue('uuid')